modular_skyrat/modules/reagent_forging/code/forge.dm

```diff
@@ -78,6 +78,40 @@
 	///list containing cooking radial buttons, for when anything of /obj/item/food is used on the forge
 	var/static/list/radial_options = list("oven" = radial_oven, "microwave" = radial_microwave)
 
+    ///Blacklist that contains reagents that weapons and armor are unable to be imbued with.
+	var/static/list/disallowed_reagents = typecacheof(list(
+		/datum/reagent/inverse/,
+		/datum/reagent/consumable/entpoly,
+		/datum/reagent/pax,
+		/datum/reagent/consumable/liquidelectricity/enriched,
+		/datum/reagent/toxin/heparin,
+		/datum/reagent/teslium,
+		/datum/reagent/eigenstate,
+		/datum/reagent/drug/bath_salts,
+		/datum/reagent/drug/methamphetamine,
+		/datum/reagent/drug/pcp,
```

Comment:
if people make it and get enough of this, they should be allowed to infuse it-- remove these.